app/src/main/java/org/oppia/android/app/testing/LessonThumbnailImageViewTestActivity.kt

```diff
@@ -0,0 +1,19 @@
+package org.oppia.android.app.testing
+
+import android.os.Bundle
+import org.oppia.android.R
+import org.oppia.android.app.activity.InjectableAppCompatActivity
+
+/** Test-only activity for verifying behaviors of LessonThumbnailImageView. */
```

Comment:
```kt
import org.oppia.android.app.customview.LessonThumbnailImageView

/** Test-only activity for verifying behaviors of [LessonThumbnailImageView]. */
```

This way we can directly jump to LessonThumbnailImageView.

Reply:
Done